### PR TITLE
Features/card

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Created different variants of card component 
1. **Horizontal card**: text+image card with text overlay and without text overlay
2. **Vertical card**: text+image card with badge and dismiss
Screenshot (20)

### CODE REVIEW

**Review by sanjay-xdr — APPROVED** [review]
Everthing looks good

**Review by xii-vi — APPROVED** [review]
Everything is in place. You can merge.

File: Components/button.css [hunk 1]
```diff
@@ -82,7 +82,12 @@
   color: var(--primary-color);
 }
 .bttn-float-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (amirnathani9) [inline Components/button.css:87]: give the difeerent name for flex classes and reuse it wherever required

File: Components/card.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+.card-horizontal {
+  display: flex;
+  background-color: var(--complimentary-text-color);
+  max-width: 31.25rem;
+  text-align: left;
+}
+.card-horizontal-content-container {
+  display: flex;
+  flex-direction: column;
```
Comment (amirnathani9) [inline Components/card.css:67]: same here u can review it

**Comment by amirnathani9** [comment]
Looks everything good. Do the changes as commented

**Pull request merged**